README.md

```diff
@@ -316,13 +317,13 @@ func getNextAudioFrame() -> UnsafeMutablePointer<Int16> {
 
 while true {
     let pcm = getNextAudioFrame()
-    var keyword_index: Int32 = -1
```

Comment:
Do you think we can just replace "pv_porcupine_process" with "pv_porcupine_multiple_keywords_process"? Then it is coherent with how we initialized the object to do multi-keyword detection above. Thank you.